src/ImageSharp/PixelFormats/PixelOperations{TPixel}.PixelBenders.cs

```diff
@@ -22,25 +22,30 @@ internal virtual PixelBlender<TPixel> GetPixelBlender(PixelBlenderMode mode)
         {
             switch (mode)
             {
-                case PixelBlenderMode.Multiply:
-                    return DefaultMultiplyPixelBlender<TPixel>.Instance;
-                case PixelBlenderMode.Add:
-                    return DefaultAddPixelBlender<TPixel>.Instance;
-                case PixelBlenderMode.Substract:
-                    return DefaultSubstractPixelBlender<TPixel>.Instance;
-                case PixelBlenderMode.Screen:
-                    return DefaultScreenPixelBlender<TPixel>.Instance;
-                case PixelBlenderMode.Darken:
-                    return DefaultDarkenPixelBlender<TPixel>.Instance;
-                case PixelBlenderMode.Lighten:
-                    return DefaultLightenPixelBlender<TPixel>.Instance;
-                case PixelBlenderMode.Overlay:
-                    return DefaultOverlayPixelBlender<TPixel>.Instance;
-                case PixelBlenderMode.HardLight:
-                    return DefaultHardLightPixelBlender<TPixel>.Instance;
+                case PixelBlenderMode.Multiply: return DefaultPixelBlenders<TPixel>.Multiply.Instance;
+                case PixelBlenderMode.Add: return DefaultPixelBlenders<TPixel>.Add.Instance;
+                case PixelBlenderMode.Substract: return DefaultPixelBlenders<TPixel>.Substract.Instance;
+                case PixelBlenderMode.Screen: return DefaultPixelBlenders<TPixel>.Screen.Instance;
+                case PixelBlenderMode.Darken: return DefaultPixelBlenders<TPixel>.Darken.Instance;
+                case PixelBlenderMode.Lighten: return DefaultPixelBlenders<TPixel>.Lighten.Instance;
+                case PixelBlenderMode.Overlay: return DefaultPixelBlenders<TPixel>.Overlay.Instance;
+                case PixelBlenderMode.HardLight: return DefaultPixelBlenders<TPixel>.HardLight.Instance;
+                case PixelBlenderMode.Src: return DefaultPixelBlenders<TPixel>.Src.Instance;
```

Comment:
Yeah.. I believe this covers all the standard composition/blending types. We can worry about making it pluggable later.